tools/update_changelog.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import re
+
+import m2r2
+import requests
+
+
+TEMPLATE = """
+.. This is a generated file. Do not edit it manually, because it is updated
+   automatically via tools/update_changelog.py.
+
+.. include:: /shortcuts.rstext
+
+.. _appendix-version-history:
+
+{content}
+
+.. seealso:: For an overview of previous versions, `take a look
+             <https://launchpad.net/mixxx/+series>`_ at the timeline.
+"""
+
+
+def fetch_changelog(branch):
+    r = requests.get(
+        "https://raw.githubusercontent.com/mixxxdj/mixxx/"
+        f"{branch}/CHANGELOG.md"
+    )
+    r.raise_for_status()
+    return r.text
+
+
+def changelog_to_rst(changelog):
+    # Replace headline with "Version History"
+    changelog = re.sub(
+        "^# Changelog$",
```

Comment:
```suggestion
        r"^#\s*Changelog\s*$",
```
makes it a bit more robust without producing false positives